common/djangoapps/student/views.py

```diff
@@ -2629,7 +2629,8 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
         # If password reset was unsuccessful a template response is returned (status_code 200).
         # Check if form is invalid then show an error to the user.
         # Note if password reset was successful we get response redirect (status_code 302).
-        if response.status_code == 200 and not response.context_data['form'].is_valid():
+        form_valid = response.context_data['form'].is_valid() if response.context_data['form'] else False
+        if response.status_code == 200 and not form_valid:
             response.context_data['err_msg'] = _('Error in resetting your password. Please try again.')
```

Comment:
Can you please log some warning here to make sure this condition caught in our splunk?